cache/src/commonMain/kotlin/org/mobilenativefoundation/store/cache5/Identifiable.kt

```diff
@@ -1,5 +1,6 @@
 package org.mobilenativefoundation.store.cache5
 
+@Deprecated("Use StoreMultiCache instead of MultiCache")
 interface Identifiable<Id : Any> {
```

Comment:
Curious, do store items need to implement identifiable and StoreData? Would it make sense to make the ID a container around data rather than having to leak Identifiable interface into people's impls? For example what happens if I want to store something like String and cannot add interface to it?